alpa/pipeline_parallel/compile_executable.py

```diff
@@ -179,9 +180,16 @@ def compile_pipeshard_executable(fun: lu.WrappedFun,
         schedule=schedule,
         is_batch=batch_invars,
         num_batch=num_microbatch,
-        flop_count=total_flops,
-        concat_vars_mapping=concat_vars_mapping,
-        in_tree=in_tree)
+        in_tree=in_tree).compile(concat_vars_mapping)
```

Comment:
Why is `concat_vars_mapping` an argument of `compile` instead of an argument of `__init__`?

Reply:
You're right. I move it to init now